tests/contrib/sanic/test_sanic.py

```diff
@@ -0,0 +1,204 @@
+import re
```

Comment:
Can we add a test for overriding the service name? It'd also be good to test to ensure that sanic will use `DD_SERVICE`

Reply:
Added tests for default service, for config override both manually and by use of environment variable.